apps/server/src/modules/authentication/strategy/jwt.strategy.spec.ts

```diff
@@ -55,39 +45,48 @@ describe('jwt strategy', () => {
 
 	describe('when authenticate a user with jwt', () => {
 		const setup = () => {
+			const mockJwtPayload = jwtPayloadFactory.build();
```

Comment:
same like in aw-jwt.strategy.spec.ts

Reply:
the review comment is to add factory and move this to setup methods